### PR TITLE
Update manage-resource-accounts.md

### PR DESCRIPTION
**Pull request by waseemhashem**
@jambirk

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-resource-accounts.md [hunk 1]
```diff
@@ -44,14 +44,15 @@ To get started it's important to remember a few things:
...
->
-> User (subscriber) phone numbers can't be assigned to a resource account. Only service toll or toll-free phone numbers can be used.
->
-> If you are outside the United States, you can't use the Microsoft Teams admin center to get service numbers. Go to [Manage phone numbers for your organization](manage-phone-numbers-for-your-organization/manage-phone-numbers-for-your-organization.md) instead to see how to do it from the outside of the United States.
+
+To redirect calls to people in your organization who are homed Online, they must have a **Phone System** license and be enabled for Enterprise Voice or have Office 365 Calling Plans. See [Assign Microsoft Teams licenses](assign-teams-licenses.md). To enable them for Enterprise Voice, you can use Windows PowerShell. For example run:  `Set-CsUser -identity "Amos Marble" -EnterpriseVoiceEnabled $true`
```
Comment (JohanFreelancer9) [inline Teams/manage-resource-accounts.md]:
```suggestion
To redirect calls to people in your organization who are homed Online, they must have a **Phone System** license and be enabled for Enterprise Voice or have Office 365 Calling Plans. See [Assign Microsoft Teams licenses](assign-teams-licenses.md). To enable them for Enterprise Voice, you can use Windows PowerShell. For example run: `Set-CsUser -identity "Amos Marble" -EnterpriseVoiceEnabled $true`
```

**Comment by scanum** [comment]
@officedocsbot assign @jambirk

**Comment by scanum** [comment]
@jambirk Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**